feed/tests/test_feed_views.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+from feed.models import Post
+from django.contrib.auth.models import User
+
+
+USERNAME = 'username'
+USER_PASS = 'secret_pass'
+POST_TITLE = 'post title'
+POST_CONTENT = 'some post content'
+POST_DETAIL_URL = '/post/'
+
+
+@pytest.fixture
+def user(db):
+    user = User.objects.create_user(USERNAME, password=USER_PASS)
+    return user
+
+
+@pytest.fixture
+def post(db, user):
+    post = Post.posts.create(title=POST_TITLE, content=POST_CONTENT, author=user)
+    post.save()
+    return post
+
+
+@pytest.mark.django_db
+class TestPostDetailView:
+    def test_detail_view_page_entrypoint(self, client):
+        # Testing to see if a valid post id gets a valid detail view page
+        post = Post.posts.main_feed().first()
```

Comment:
This should be returned be created with fixture post, 
Once post is returned, from fixture, you can call str(post.id)
Try to maintain only client logic here

Reply:
Done